crates/vm/levm/src/vm.rs

```diff
@@ -324,7 +324,7 @@ impl VM {
             // If address is already in db, there's an error
             let new_address_acc = self
                 .db
-                .get_account_info(self.call_frames.first().unwrap().to);
+                .get_account_info(self.call_frames.first().ok_or(VMError::FatalUnwrap)?.to);
```

Comment:
This could be an internal error `VMError::Internal(InternalError::CouldNotAccessFirstCallframe)`, following this suggested hierarchy https://github.com/lambdaclass/lambda_ethereum_rust/pull/1107#pullrequestreview-2427589737.

Reply:
https://github.com/lambdaclass/lambda_ethereum_rust/pull/1108/commits/c7a293ab826b8514b8142b41401a2e943a16cfc1